packaging/packager_win.bat

```diff
@@ -18,15 +18,20 @@ SET DISABLE_VNCDOTOOL=1
 REM Main deps
 cd %DISTRO_ROOT%\
 REM Assuming a local python executable to avoid overloading the download web page
-START /WAIT python-3.9.0-amd64.exe /quiet InstallAllUsers=1 PrependPath=1 Include_test=0
+START /WAIT python-3.12.6-amd64.exe /quiet InstallAllUsers=1 PrependPath=1 Include_test=0
 REM set a temporary path variable valid inside this session
-set PYTHONPATH="C:\Program Files\Python39"
+set PYTHONPATH="C:\Program Files\Python312"
 set PATH=%PYTHONPATH%;%PYTHONPATH%\Scripts;%PATH%
 REM a permanent path variable will be set by the executable once this batch exits
+REM Download the Visual C++ Redistributable package

```

Comment:
I assume the C++ executable and setuptools are needed to build some dependency? We could maybe add a clarification in a comment.

Reply:
Okay